package.json

```
@@ -54,9 +54,9 @@
     "test:e2e": "playwright test",
     "test:unit": "vitest",
     "test:unit:ci": "vitest run",
-    "lint": "eslint . --max-warnings 0",
-    "lint:fix": "eslint . --fix",
-    "format": "prettier --write './**/*.{js,jsx,ts,tsx,css,md,json}' --config ./.prettierrc",
+    "lint": "eslint \"src/**/*.{js,ts,jsx,tsx}\" --quiet",
```

```suggestion
    "lint": "eslint \"src/**/*.{js,ts,jsx,tsx}\"",
```
I think we'll want the warnings
additionally, the glob patterns should be placed in the [`eslintrc` file](https://github.com/hasadna/open-bus-map-search/blob/main/.eslintrc.json)
[documentation](https://eslint.org/docs/latest/use/configure/configuration-files-new#excluding-files-with-ignores)